module/VuFind/src/VuFind/SMS/Clickatell.php

```diff
@@ -171,9 +171,10 @@ protected function formatMessage($message)
     {
         // Clickatell expects UCS-2 encoding:
         if (!function_exists('iconv')) {
-            // @codeCoverageIgnoreStart
-            throw new MailException('Clickatell requires iconv PHP extension.');
-            // @codeCoverageIgnoreEnd
+            throw new MailException(
+                'Clickatell requires iconv PHP extension.',
+                MailException::ERROR_EXTENSION_MISSING
```

Comment:
This seems like a more technical error, and not something meaningful to the end user -- maybe this should just stick with the UNKNOWN code.

Reply:
Check